src/styles/RTLHelper.js

```diff
@@ -0,0 +1,2 @@
+// eslint-disable-next-line import/prefer-default-export
+export const isRTL = () => document.dir === 'rtl';
```

Comment:
One more thing, actually. Are we sure the `dir` attribute will be set?
There's also the `Intl` API we could use instead:
```
const intl = new Intl.Locale(navigator.language);
const isRtl = intl.textInfo.direction === 'rtl';
```